shared/utils/__tests__/logic.spec.ts

```diff
@@ -956,6 +937,8 @@ describe('Logic validation', () => {
           _id: MOCK_LOGIC_ID,
           logicType: LogicType.PreventSubmit,
         } as unknown as FormLogic,
+        // This requires to be cast to unknown first at it requires _id property,
+        // which was previously inherited from IPreventSubmitLogicSchema
```

Comment:
This is a code smell. Does this mean that we actually need the `_id`  parameter?

Reply:
yup it seems so, the test cases in the latest commit are also failing due to the removal of `_id` also. Should I revert to the previous commit in this case

Reply:
No, this means our types are wrong and _id is actually needed. Please fix the types instead of ignoring it! 🙏